hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/libhdfspp/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ project (libhdfspp)
 
 cmake_minimum_required(VERSION 2.8)
 
-find_package (Boost 1.72.0 REQUIRED)
+find_package (Boost 1.72.0 REQUIRED COMPONENTS date_time)
```

Comment:
Is there a way to make this 1.72.0 a variable?
I'd like to avoid setting this in 4 places.

Reply:
> I'd like to avoid setting this in 4 places.
I've now removed the redundant calls to `find_package(Boost ...)`. I'm calling it only in the main `CMakeLists.txt` now.